### PR TITLE
Add cross platform support for Windows

### PR DESCRIPTION
**Pull request by Robdel12**
## What is this?

This adds support for running the example app from windows. I have verified this works with windows with this specific build coming from my windows machine: https://percy.io/percy/example-percy-cypress/builds/1564784

image

### CODE REVIEW

File: run-tests.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const spawn = require('child_process').spawn;
+const platform = require("os").platform();
+
+// We need to change the command path based on the platform they're using
+const cmd = /^win/.test(platform)
+      ? `${process.cwd()}\\node_modules\\.bin\\cypress.cmd`
+      : `cypress`;
+
+const server = httpServer.createServer()
+const PORT = 8000
```
Comment (Robdel12) [inline run-tests.js]: This isn't used

File: package.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
   "devDependencies": {
     "@percy/cypress": "^1.0.2",
     "cypress": "^3.1.5",
-    "http-server": "^0.11.1"
+    "http-server": "^0.11.1",
+    "https-server": "^0.1.2"
```
Comment (anaulin) [inline package.json]: Do you ever use this `https-server` package? Might not be needed.

Comment (Robdel12) [reply package.json]: Very good catch! I did not intend to add that 😱

**Comment by anaulin** [comment]
All builds are 💚 now, so feel free to merge as soon as you remove that `https-server` dependency. :shipit:

**Pull request merged**